devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
+
+(1) The catalogs themselves are in AEGIS_galaxy_catalog_[VI]_25.2.fits (where
+[VI] indicates either V or I).  These files can be used to initialize a
```

Comment:
I think I slightly prefer having separate (but aligned) catalogs for V and I.  For one thing, it's backwards compatible with `RealGalaxyCatalog`.

The way I've currently set up `ChromaticRealGalaxy` is to take a list of catalogs as one of its input parameters:

```python
crg = galsim.ChromaticRealGalaxy([Vcat, Icat], index=123)
```

Part of the motivation here was thinking ahead to a CANDELS catalog, where we might want to test the difference between using all 4 bands as input to using only 2 or 3 bands as input.  This is trivial with the above API and independent catalogs.